pom.xml

```diff
@@ -271,6 +276,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-enforcer-plugin</artifactId>
+                <version>1.4.1</version>
```

Comment:
Was the latest release (3.5.0) unsuccessful?

Reply:
Good catch.